Jenkinsfile_nightly

```diff
@@ -84,7 +84,6 @@ withNightlyPipeline(type, product, component) {
   enableFullFunctionalTest(240)
   // 'microsoft' and 'safari' to be returned to crossbrowser tests once FPLA-3172 improvements are complete
   enableCrossBrowserTest(['chrome', 'firefox'])
-  enableFortifyScan()
```

Comment:
do we want to get rid of the fortify scan on nightly? I don't think we run it anywhere else as far as I can see?

Reply:
Wasn't Fortify for static code analysing which is required.

Reply:
true, i think we need to keep this. I'll add it back in